src/style.css

```diff
@@ -1,3 +1,86 @@
+
 body {
   margin: 0;
+  padding: 0;
+  background-color: #ebebe4;
+  font-family: Roboto, Inter, Helvetica, sans-serif;
+  box-sizing: border-box;
+  --color-selected: #00acdc;
+}
+
+.header {
+  display: flex;
+  margin: auto;
+  justify-content: space-between;
+  align-items: center;
+  min-width: 1200px;
+  padding: 0 50px;
+  background-color: #fff;
+  box-sizing: border-box;
+}
+
+.header__logo {
+  display: block;
+  width: 40px;
+  height: 40px;
+}
+
+.logo {
+  display: inline-block;
+  text-decoration: none;
+  background-color: red;
+}
+
+.nav__list {
+  margin: 0;
+  padding: 0;
+  display: flex;
+}
+
+.nav__list :first-child {
+  margin: 0;
+}
+
+.nav__item {
+  position: relative;
+  list-style: none;
+  margin-left: 20px;
+}
+
+.nav__link {
+  display: flex;
+  flex-direction: column;
+  line-height: 60px;
+  text-decoration: none;
+  text-transform: uppercase;
+  cursor: pointer;
+  font-size: 12px;
+  font-style: normal;
+  font-weight: 500;
+  color: #000;
+}
+
+.is-active {
+  color: var(--color-selected);
+}
+
+.is-active:after {
+  display: block;
+  position: absolute;
+  bottom: 0;
+  content: "";
+  height: 4px;
+  width: 100%;
+  border-radius: 8px;
+  background-color: var(--color-selected);
+}
+
+.nav__link:hover {
+  color: var(--color-selected);
+}
+
+@media (max-width: 1024px) {
```

Comment:
This rule says that for screen sizes that smaller than 1024px we apply css rule for the element with class name header that makes header to be at least 1024px. You are trying to insert bigger element into smaller screen. Rewrite this or remove after fixing the previous comment related to width: 100% + max-width

Reply:
if you narrow the screen any further, it breaks the layout, so I think that 

@media (max-width: 1024px) {
  .header {
    width: 1024px;
  }
}

still need as minimum size 
changed min-width: to width: 1024px;
or delete this @media completely after all

Reply:
This rule is an overkill for this task